core/components/molecules/sidesheet/__stories__/stickyFooter.story.jsx

```diff
@@ -113,24 +110,21 @@ const customCode = `() => {
   };
 
   const modalDescriptionOptionsWithoutTitle = {
-    description: 'Card Sections include supporting text like an article summary or a restaurant description.',
+    description: 'Card Sections include supporting text like an article summary or a healthcare service description.',
     removePadding: true
   };
 
   return (
     <div>
       <Button appearance="primary" onClick={() => setOpen(true)}>Open Sidesheet</Button>
       <Sidesheet {...options} >
-        <Text>Modal Body</Text>
         <ModalDescription {...modalDescriptionOptions} />
         <ModalDescription {...modalDescriptionOptionsWithoutTitle} />
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit,
-        sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.<br />
-				Ut enim ad minim veniam,
-        quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.<br />
-				Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. <br />
-				Excepteur sint occaecat cupidatat non proident,
-        sunt in culpa qui officia deserunt mollit anim id est laborum.<br />
+        <p>
+          The action buttons appear in the left side of the sheet and follows the reverse order as the modals to keep
+          the position of primary action predictable. Also, if the content covers &gt;= 75% of the side sheet measured
+          vertically, then the actions appear in the bottom, i.e, margin-bottom for the footer is 0px.
+        </p>
```

Comment:
Please use `Paragraph` component instead of using `p` tag

Reply:
Done